azkaban-webserver/src/test/java/azkaban/webapp/servlet/ExternalAnalyzerUtilsTest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2016 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package azkaban.webapp.servlet;
```

Comment:
This doesn't seem to assert the result link is generated correctly.

And the return value of the method seems to be only used for this testing purposes.

See my comment in the production code about a possible alternative.